venues/ICLR.cc/2019/Conference/webfield/authorWebfield.js

```diff
@@ -252,6 +252,10 @@ function renderContent(notes, submittedNotes, assignedNotePairs, assignedNotes,
     if (!_.includes(COMMENT_EXCLUSION, note.invitation)) {
       commentNotes.push(note);
     }
+    if (note.invitation === SUBMISSION_ID &&
+        !_.includes(_.map(authorNotes, n=>n.id), note.id)) {
```

Comment:
So unfortunately in Webfield code we can't use any ES6 features, so => needs to be replace with a regular function.

Reply:
@zbialecki I actually see the => functions in a bunch of other places in the code - is that bad?

Reply:
If it's in webfield files, yes.

Reply:
It will cause JS errors in all older browsers.